scripts/alerts-during-perf-testing.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# USAGE
+# ./alerts-during-perf-testing.sh <optional product-name>
+# ^C to break
+# Generates two files one for firing and one for pending alerts
+#
+# PREREQUISITES
+# - jq
+# - oc (logged in at the cmd line in order to get the bearer token)
+
+# remove tmp files on ctrl-c
+trap "rm tmp-alert-firing-during-perf-testing-report.csv tmp-alert-pending-during-perf-testing-report.csv" EXIT
+
+# function to check if there are no alerts firing bar deadmansnitch
+function CHECK_NO_ALERTS(){
+    if [[ $(curl -s -H "Authorization: Bearer $(oc whoami --show-token)" $(echo "https://$(oc get route prometheus-route -n redhat-rhmi-middleware-monitoring-operator -o=jsonpath='{.spec.host}')")/api/v1/alerts | jq -r '.data.alerts[]| select(.state=="firing") | [.labels.alertname, .state, .activeAt ] | @csv' | wc -l) == 1 ]] ; then
```

Comment:
can we assign the token to some variable?

Reply:
:+1: